PSICover/PSICover.cs

```diff
@@ -165,9 +238,13 @@ class Analyzer {
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
          foreach (var block in blocks) {
             bool hit = hits[block.Id] > 0;
-            string tag = $"<span class=\"{(hit ? "hit" : "unhit")}\">";
-            code[block.ELine] = code[block.ELine].Insert (block.ECol, "</span>");
-            code[block.SLine] = code[block.SLine].Insert (block.SCol, tag);
+            string tag = $"<span class=\"{(hit ? "hit" : "unhit")}\" title = \"Hits: {hits[block.Id]}\">";
+            for (int i = block.ELine; i >= block.SLine; i--) {
+               int mInsert = (i == block.ELine) ? mInsert = block.ECol : code[i].Length;

```

Comment:
mInsert = is not required before block.ECol. Variable name should not start with mI as it is not class member.

Reply:
Corrected